### ISSUE TITLE
Add a section about micro-services

### ISSUE THREAD

**Issue by blittle** [CanopyTax/frontend-rfc#2]

**Comment by blittle** [comment]
This probably could include information about sofe, single-spa, common-dependencies and patterns for sharing data between services.

**Comment by blittle** [comment]
Also common dependencies

**Comment by TheMcMurder** [comment]
Are you envisioning supported patterns for sharing data and the boundaries between Autonomy and Alignment or something much looser?

**Comment by blittle** [comment]
Yeah, I think we need to document that single-spa and sofe are very official, currently you can't really create a new bit of functionality without those. With that restriction, what are common pitfalls and strategies to overcome them?
